.bazelrc

```diff
@@ -47,6 +47,17 @@ build --host_per_file_copt='external/zlib@-Wno-unknown-warning-option,-Wnodeprec
 # for some reason tcmalloc uses internal msan-enabled annotation in the code, rather than public one
 build --per_file_copt=external/com_google_tcmalloc@-DANNOTATE_MEMORY_IS_INITIALIZED=ABSL_ANNOTATE_MEMORY_IS_INITIALIZED
 
+# Increasing the optimization level of some portions of V8 significantly speeds up Python tests
```

Comment:
I know I suggested this approach, but it should be explained well here – it should be clear that this is an optional "performance hack" and we could explore if applying this to only a few folders is sufficient to get most of the speedup for python. This is not a blocking issue however, V8 is cached for the vast majority of builds so build speeds should be acceptable. – I hope that we'll reduce the amount of code we need to compile for python soon so that there's less of a need for manual intervention here.

We can't apply this by default unfortunately – the release configuration uses -O3 so using -O2 here reduces the optimization level for these files. One workaround would be to define a build:v8-codegen-opt configuration and add --config=v8-codegen-opt to `bazelisk build [...]` and `bazelisk test [...]` in .github/workflows/test.yml – it is safe to apply this to all debug builds.

Before merging, make sure that the downstream branch has tests passing to avoid any breakage there.